local-tests/chainrunner/chain.py

```diff
@@ -0,0 +1,108 @@
+import os
+import os.path as op
+import subprocess
+
+from .node import node
+from .utils import flags_from_dict
+
+
+# Type alias for int
+class Seq(int): pass
```

Comment:
Because then `isinstance(3,Seq) == True` and there would be no way to distinguish constant int parameters vs the ones that should be different for each node.

Reply:
fair, but then I have a problem with the term 'alias'; pls change it to wrapper or smth more relevant

Reply:
Good point, done